### PR TITLE
Add https as valid protocol

### PR DESCRIPTION
**Pull request by bulvestad**
hope its ok to add https as valid protocol as default

### CODE REVIEW

**Comment by bulvestad** [comment]
@geiralund @michaelsembwever Unsure howto invite someone to review pullrequests on github :-) Hopefully this helps. I hope this change is ok?

File: src/test/java/net/sf/xsshtmlfilter/HTMLFilterTest.java [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
```
Comment (michaelsembwever) [inline src/test/java/net/sf/xsshtmlfilter/HTMLFilterTest.java]: is there any need to change the import order?

**Comment by michaelsembwever** [comment]
Looks good to me.

Could you please
- squash the commits into one,
- undo the change in import statement order in HtmlFilterTest

File: src/test/java/net/sf/xsshtmlfilter/HTMLFilterTest.java (hunk 1 above)
Comment (bulvestad) [inline src/test/java/net/sf/xsshtmlfilter/HTMLFilterTest.java]: No, IntelliJ did it for me and I didn't see it before the commit was done. 
The order is the one we agreed upon in FINN (unless I missed something?), and I though I might as well leave it unless you objected. Do you want me to change it back? The first import import net.sf.xsshtmlfilter.HTMLFilter isn't necessary and can be ommitted.

Comment (michaelsembwever) [inline src/test/java/net/sf/xsshtmlfilter/HTMLFilterTest.java]: you can't be enforcing a private and unbeknown to all others code style.

in lei of any documented code style for this project, and any precedence in its other java files, i would simply avoiding changes in pull requests that are unnecessary. that's a pretty normal expectation in the open source world.

> The first import import net.sf.xsshtmlfilter.HTMLFilter isn't necessary and can be ommitted.

Yes that can (should) be omitted.

**Comment by bulvestad** [comment]
@michaelsembwever Should I feel free to release this project myself, or is it something you want to control? And if I get to release is it the standard finn way with mvn:release ?

**Comment by geiralund** [comment]
Nice, thanks @bulvestad :)

**Comment by geiralund** [comment]
I will delete the branch.

**Comment by bulvestad** [comment]
After discussing with @geiralund I tried to release this, but wasn't able to get it done. 
Anyone who knows how? @michaelsembwever ?

**Comment by michaelsembwever** [comment]
this project doesn't have the maven-release-plugin configured, so you need to make manual releases.

change the pom to the release version, commit, tag, change pom to next dev version, commit.

Then locally checkout the release tag you just made do 
 a) run `mvn package` and upload the built artifact into finn's nexus, and
 b) go make a "github release" uploading the same built artifact.

**Pull request merged**